lib/babel-istanbul-instrumenter.js

```diff
@@ -0,0 +1,256 @@
+/**
```

Comment:
No offense taken, but my requests here seem reasonable (perhaps only to me?) and have been communicated very clearly (for most of the time this PR has been waiting). As I mentioned, a number of times before, once this is addressed, the bulk of the work will be merged and we can have a specific conversation about this file on its own merit.